### ISSUE TITLE
ctrl-click on contact from contact list

### ISSUE THREAD

**Issue by bbrendon** [monicahq/monica#3376]
If I go somewhere and I see a few people, it's a lot of clicking to edit all of them. If I could just ctrl-click the ones from the list to make new browser tabs and then edit them in the new tabs, it would be a _lot_ faster to make changes.

**Comment by asbiin** [comment]
Hi. Can you explain where this is not working?

**Comment by bbrendon** [comment]
Notice in the screenshot there is no "open link in new tab". When you right click on a contact.

![image](https://user-images.githubusercontent.com/6364477/71539883-ff17bb80-28f7-11ea-9800-32db7bd6c3c9.png)

**Comment by bbrendon** [comment]
This is a feature request really. Not a bug.

**Comment by asbiin** [comment]
OK I get it: it's working on the avatars, but not on the other cells of the table ...

PS: it seems you have not set a queue worker to create the default avatars.

**Comment by bbrendon** [comment]
Hmm. I didn't realize clicking on the Avatar would have a different result. I tried it and got some odd behavior. (Chrome).

1. When I first tried it, I would see popup-blocked in the broswer... so I enabled popups in advanced settings for the site.

2. Tried it again, no more "blocked" message but when I ctrl-click an avatar from the contact list I end up getting two browser tabs of the person I ctrl-clicked. The original tab goes to the contact and I get a new tab with the contact.
